README.md

```diff
@@ -2,4 +2,95 @@
 
 # Awesome FSRS [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
 
- A curated list of awesome FSRS implementations, papers and resources. Feel free to suggest new projects in Issues or PR directly.
+A curated list of awesome FSRS implementations, papers and resources. Feel free to suggest new projects in Issues or PR directly.
+
+## Implementation
+
+- Python
+  - Scheduler (v5): [py-fsrs](https://github.com/open-spaced-repetition/py-fsrs)
+  - Optimizer: [fsrs-optimizer](https://github.com/open-spaced-repetition/fsrs-optimizer)
+  - [Deprecated] Optimizer: [fsrs-optimizer-tiny](https://github.com/open-spaced-repetition/fsrs-optimizer-tiny)
+- Rust
+  - Scheduler (v5) + Optimizer: [fsrs-rs](https://github.com/open-spaced-repetition/fsrs-rs)
+    - Run in browsers: [fsrs-browser](https://github.com/open-spaced-repetition/fsrs-browser)
+- TypeScript
+  - Scheduler (v5): [ts-fsrs](https://github.com/open-spaced-repetition/ts-fsrs)
+- Go
+  - Scheduler (v5): [go-fsrs](https://github.com/open-spaced-repetition/go-fsrs)
+- Dart
+  - Scheduler (v4.5): [dart-fsrs](https://github.com/open-spaced-repetition/dart-fsrs)
+- Swift
+  - Scheduler (v4.5): [swift-fsrs](https://github.com/open-spaced-repetition/swift-fsrs)
+- Clojure/ClojureScript
+  - Scheduler (v4): [cljc-fsrs](https://github.com/open-spaced-repetition/cljc-fsrs)
+- Ruby
+  - Scheduler (v4): [rb-fsrs](https://github.com/open-spaced-repetition/rb-fsrs)
+- Kotlin
+  - Scheduler (v4): [android-fsrs](https://github.com/open-spaced-repetition/android-fsrs)
+- Elixir
+  - Scheduler (v4): [ex_fsrs](https://github.com/open-spaced-repetition/ex_fsrs)
+
+## Applications
+
+### Flashcard
+
+- #### Anki (https://apps.ankiweb.net/)
+
+  Free and open source content-agnostic flashcard application for Windows, Mac, Linux, iPhone, and Android. Supports images, audio, videos, and scientific markup (via LaTex). Offers free synchronization service using AnkiWeb, with community shared add-ons and decks.
+
+  FSRS available as opt-in feature replacing the default SM-2 algorithm.
+
+- #### MaiMemo (https://www.maimemo.com/)
+
+  Chinese mobile flashcard application for studying English.
```

Comment:
Any comments on MaiMemo description? @L-M-Sherlock 
I can't find much information about the application without using it.